hot-reload-runtime-jvm/src/main/kotlin/org/jetbrains/compose/reload/jvm/window.kt

```diff
@@ -33,121 +27,124 @@ import java.awt.event.WindowEvent
 
 private val logger = createLogger()
 
-@Composable
+private var curWindowId: WindowId? = null
+private var curWindow: Window? = null
```

Comment:
There can be multiple windows, so I think this code is violating that assumption. 
As a second note: I think holding a strong reference to an object like window, statically might be a bad idea.